### PR TITLE
[#31037] Add various Drupal elements.

### PR DESCRIPTION
**Pull request by HierenFeb**
test it please

### CODE REVIEW

**Comment by HierenFeb** [comment]
test this please

File: themes/agov/agov_whitlam/sass/components/progress-bar/progress-bar.hbs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div id="progress" class="progress">
+  <div class="bar"><div class="filled" style="width: 60%"></div></div>
+  <div class="percentage">60%</div>
+  <div class="message">Installing...</div>
+</div>
```
Comment (kimpepper) [inline themes/agov/agov_whitlam/sass/components/progress-bar/progress-bar.hbs]: Always put a single newline at the end of files.

File: themes/agov/agov_whitlam/sass/components/highlight-mark/highlight-mark.hbs [hunk 2]
```diff
@@ -0,0 +1 @@
+<mark class="highlight-mark">New</mark>
```
Comment (kimpepper) [inline themes/agov/agov_whitlam/sass/components/highlight-mark/highlight-mark.hbs]: Always put a single newline at the end of files.

File: themes/agov/agov_whitlam/sass/components/progress-bar/progress-bar.hbs (hunk 1 above)
Comment (HierenFeb) [inline themes/agov/agov_whitlam/sass/components/progress-bar/progress-bar.hbs]: Thanks! I inserted it!

File: themes/agov/agov_whitlam/sass/components/highlight-mark/highlight-mark.hbs (hunk 2 above)
Comment (HierenFeb) [inline themes/agov/agov_whitlam/sass/components/highlight-mark/highlight-mark.hbs]: Thanks! I inserted and remember it!

**Comment by JohnAlbin** [comment]
Reviewing…

**Pull request merged**